Library/Homebrew/brew.rb

```diff
@@ -10,7 +10,8 @@
 
 require "pathname"
 HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent
-$:.unshift(HOMEBREW_LIBRARY_PATH.to_s)
+require "English"
```

Comment:
We are probably requiring this somewhere else down the line, these can be removed now that we have it at the very beginning.